astronomy/epoch.hpp

```diff
@@ -36,17 +36,22 @@ CONSTEXPR_INFINITY Instant InfiniteFuture = J2000 + Infinity<Time>();
 // The Julian Date JD |days|. J2000.0 is JD 2451545.0. |days| is the number of
 // days since -4712-01-01T12:00:00,000 (Terrestrial Time, Julian calendar).
 constexpr Instant JulianDate(double days);
+constexpr double JulianDayNumber(Instant const& t);
```

Comment:
OK for this CL, but in relation to #1355, mark for deprecation within Catalan: the Julian Dates should really just be literals much like the ISO 8601 dates are (and this won't be needed once #1355 is fixed).